my-app/.gitignore

```diff
@@ -1,23 +0,0 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I think we probably want to keep the gitignore file. Doesn't really do much now, but when we get pipelines setup for deployment we will need it.

Reply:
I was trying to ignore package-lock.json but realized that I should include it https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json "This file is intended to be committed into source repositories, and serves various purposes" oops!